cohere/client.py

```diff
@@ -96,6 +96,7 @@ def generate(self,
                  p: float = None,
                  frequency_penalty: float = None,
                  presence_penalty: float = None,
+                 end_sequences: List[str] = None,
```

Comment:
why not include it in a test case? or do we not test every param?

Reply:
The way the tests are set up currently, it's hard to test parameters that alter the generation.